lib/src/nutripatrol/get_flag.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'get_ticket.dart';
+import '../prices/flavor.dart';
+
+import '../interface/json_object.dart';
+import '../utils/nutripatrol_types.dart';
+
+part 'get_flag.g.dart';
+
+@JsonSerializable()
+class NutripatrolFlag extends JsonObject {
+  /// Flag ID. Read-only.
+  @JsonKey()
+  late String id;
+
+  /// Ticket ID. Read-only.
+  @JsonKey()
+  late NutripatrolTicket ticket;
+
+  /// Barcode of the product. Read-only.
+  @JsonKey()
+  String? barcode;
+
+  /// Type of the flag
+  @JsonKey()
+  late NutripatrolType type;
+
+  /// Url of the product. Read-only.
+  @JsonKey()
+  late String url;
+
+  /// User ID. Read-only.
+  @JsonKey(name: 'user_id')
+  late String userId;
+
+  /// Device ID. Read-only.
+  @JsonKey(name: 'device_id')
+  late String deviceId;
+
+  /// Source of the flag
+  @JsonKey()
+  late NutripatrolSource source;
+
+  /// Confidence of the flag
+  @JsonKey()
+  double? confidence;
+
+  /// Image ID of the flag. Read-only.
+  @JsonKey(name: 'image_id')
+  String? imageId;
+
+  /// Flavor of the flag
+  @JsonKey()
+  late Flavor flavor;
+
+  /// Reason of the flag
+  @JsonKey()
+  String? reason;
+
+  /// Comment of the flag
+  @JsonKey()
+  String? comment;
+
+  /// created date of the ticket. Read-only.
+  @JsonKey(name: 'created_at')
+  late DateTime createdAt;
```

Comment:
cf. other comment about `DateTime` and `stringTimestampToDate`.
Besides, I would prefer `created`, as in the rest of the package.